VERSION

```diff
@@ -0,0 +1 @@
+1.31.1
```

Comment:
We'll need to ensure this updates the current latest version on `master/HEAD` before merging this PR.